src/scripts/main.js

```diff
@@ -1,3 +1,25 @@
 'use strict';
 
-// write your code here
+const table = document.querySelector('table');
+const th = table.tHead.rows[0].cells[1];
```

Comment:
The use of hardcoded cell indices (e.g., `cells[1]`) can lead to errors if the table structure changes. Consider using a more dynamic approach to identify the target cells.